Resources/Maps/_Mono/Shuttles/BlackMarket/kessler.yml

```
@@ -398,7 +398,20 @@
         chunkSize: 4
     - type: GasTileOverlay
     - type: RadiationGridResistance
-  - uid: 241
+  - uid: 2
+    components:
+    - type: MetaData
+    - type: Transform
+      pos: -0.5009518,-6.966812
+      parent: 1
+    - type: RadarBlip
+      enabled: False
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)